tests/e2e/lib.sh

```diff
@@ -93,6 +93,7 @@ export_test_environment() {
     ci_export ROX_SEARCH_PAGE_UI "${ROX_SEARCH_PAGE_UI:-true}"
     ci_export ROX_SYSTEM_HEALTH_PF "${ROX_SYSTEM_HEALTH_PF:-true}"
     ci_export ROX_SYSLOG_EXTRA_FIELDS "${ROX_SYSLOG_EXTRA_FIELDS:-true}"
+    ci_export ROX_VULN_MGMT_WORKLOAD_CVES "${ROX_VULN_MGMT_WORKLOAD_CVES:-true}"
```

Comment:
I'd like to have this on for e2e tests early, so that we can write Cypress tests during development instead of all at the end. This caused problems with Collections though, since that feature also required `ROX_POSTGRES_DATASTORE` which wasn't always enabled in the e2e tests.

Should we double-flag this feature in the code? Or is Postgres going to be enabled soon in all e2e tests given that it is required in the upcoming release?